HaxeManual/assets/Extension3.hx

```diff
@@ -0,0 +1,15 @@
+typedef PointX = {
+  var x:Int;
+}
+
+typedef PointY = {
+  var y:Int;
+}
+
+typedef PointXY = PointX & PointY;
```

Comment:
I wonder if this is a good example, doesn't seem like something you would do in real code? Maybe something like `typedef Point3D = Point2D & {z: Int}` would make more sense.